2021-07-29 17:54:01.475 [main] INFO  com.github.JavacLMD.projectZero.Database - Connection established to jdbc:mysql://localhost:3306/default
2021-07-29 17:54:01.481 [main] INFO  com.github.JavacLMD.projectZero.Database - Established customer table from connection: com.mysql.cj.jdbc.ConnectionImpl@5fe1ce85
2021-07-29 17:54:01.482 [main] INFO  com.github.JavacLMD.projectZero.Database - Established pet table from connection: com.mysql.cj.jdbc.ConnectionImpl@5fe1ce85
2021-07-29 20:27:34.949 [main] INFO  com.github.JavacLMD.projectZero.Database - Connection established to jdbc:mysql://localhost:3306/default
2021-07-29 20:27:34.958 [main] INFO  com.github.JavacLMD.projectZero.Database - Established customer table from connection: com.mysql.cj.jdbc.ConnectionImpl@5fe1ce85
2021-07-29 20:27:34.959 [main] INFO  com.github.JavacLMD.projectZero.Database - Established pet table from connection: com.mysql.cj.jdbc.ConnectionImpl@5fe1ce85
2021-07-29 22:28:23.853 [main] ERROR com.github.JavacLMD.projectZero.Database - Connection failed from url
No suitable driver found for url
2021-07-29 22:28:38.951 [main] ERROR com.github.JavacLMD.projectZero.Database - Connection failed from url
No suitable driver found for url
2021-07-29 22:28:46.630 [main] ERROR com.github.JavacLMD.projectZero.Database - Connection failed from url
No suitable driver found for url
2021-07-30 08:35:04.167 [main] ERROR com.github.JavacLMD.projectZero.Database - Connection failed from jdbc:mysql://localhost:3306
Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
2021-07-30 08:57:33.067 [main] ERROR com.github.JavacLMD.projectZero.Database - Connection failed from jdbc:mysql://localhost:3306
Access denied for user 'oops'@'localhost' (using password: NO)
2021-07-30 09:19:37.059 [main] ERROR com.github.JavacLMD.projectZero.Database - Connection failed from jdbc:mysql://localhost:3306
Access denied for user 'root'@'localhost' (using password: YES)
